tests/templates/kuttl/mount-dags-configmap/health.py

```diff
@@ -2,14 +2,35 @@
 import logging
 import requests
 import sys
+import time
 
 if __name__ == "__main__":
     result = 0
+
     log_level = 'DEBUG'
     logging.basicConfig(level=log_level, format='%(asctime)s %(levelname)s: %(message)s', stream=sys.stdout)
 
-    http_code = requests.get("http://airflow-webserver-default:8080/api/v1/health").status_code
+    url = "http://airflow-webserver-default:8080/api/v1/health"
+    count = 1
```

Comment:
Nit: isn't the log message in line 24 off by one if this is initialized with 1 instead of 0?

Reply:
Changed as per comment